bug: helm plugins can sometimes panic if the command isn't located

If I execute a plugin whose `command:` is pointed to a non-existent location, I get a PathError instead of an ExitError, which is probably why the Cmd.Run docs say:

> If the command fails to run or doesn't complete successfully, the error is of type *ExitError. Other error types may be returned for I/O problems.

(Note to self: Read the second sentence.)

```
⇒  helm github
panic: interface conversion: error is *os.PathError, not *exec.ExitError

goroutine 1 [running]:
panic(0x158e5c0, 0xc420514b80)
	/usr/local/Cellar/go/1.7.1/libexec/src/runtime/panic.go:500 +0x1a1
main.loadPlugins.func1(0xc4202f7b00, 0x227f890, 0x0, 0x0, 0x0, 0x0)
	/Users/mattbutcher/Code/Go/src/k8s.io/helm/cmd/helm/plugins.go:81 +0x383
k8s.io/helm/vendor/github.com/spf13/cobra.(*Command).execute(0xc4202f7b00, 0x227f890, 0x0, 0x0, 0xc4202f7b00, 0x227f890)
	/Users/mattbutcher/Code/Go/src/k8s.io/helm/vendor/github.com/spf13/cobra/command.go:571 +0x234
k8s.io/helm/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc420523d40, 0xc4202f6b40, 0xc4202f6d80, 0xc4204f06c0)
	/Users/mattbutcher/Code/Go/src/k8s.io/helm/vendor/github.com/spf13/cobra/command.go:661 +0x367
k8s.io/helm/vendor/github.com/spf13/cobra.(*Command).Execute(0xc420523d40, 0xc420030010, 0xc420523d40)
	/Users/mattbutcher/Code/Go/src/k8s.io/helm/vendor/github.com/spf13/cobra/command.go:620 +0x2b
main.main()
	/Users/mattbutcher/Code/Go/src/k8s.io/helm/cmd/helm/helm.go:141 +0x47
```

--- a/cmd/helm/plugins.go
+++ b/cmd/helm/plugins.go
@@ -78,9 +78,11 @@ func loadPlugins(baseCmd *cobra.Command, home helmpath.Home, out io.Writer) {
 				prog.Stdout = out
 				prog.Stderr = os.Stderr
 				if err := prog.Run(); err != nil {
-					eerr := err.(*exec.ExitError)
-					os.Stderr.Write(eerr.Stderr)
-					return fmt.Errorf("plugin %q exited with error", md.Name)
+					if eerr, ok := err.(*exec.ExitError); ok {
+						os.Stderr.Write(eerr.Stderr)
+						return fmt.Errorf("plugin %q exited with error", md.Name)
+					}
+					return err
 				}
 				return nil
 			},

